### ISSUE TITLE
Auto Stop Mining When Computer In Use

### ISSUE THREAD

**Issue by dmilin1** [tutulino/Megaminer#315]
I have a dedicated mining rig and a gaming PC, both of which I use for mining. However, I sometimes stop mining on my PC to play games. I often forget to restart the mining when I walk away from my computer. Is there any way to have mining auto stop/start when it detects mouse movement?

**Comment by Wylliam1220** [comment]
I'm assuming launching an auto (no input) bat file on idle for windows task scheduler would work

**Comment by 1001rapid** [comment]
@dmilin1 Create batch file with "taskkill" command for megaminer window and use Task Scheduler for start this bat file when your "rig" is unlock.
Do not distract the author from any nonsense. Use Google.)

**Comment by tutulino** [comment]
Sorry not available and not in roadmap.

**Comment by dmilin1** [comment]
I created a solution for myself using AutoHotKey. I've attached it here for anyone else who may find it useful. There are two versions. An uncompiled ahk script for those who might want to modify it for themselves or don't trust me and want to see the code, and a compiled exe for those who want something simple. Drop your preferred version into the root of the Megaminer folder to use it.

The program will automatically run the START.bat file if it detects the mouse position is the same as it was 5 minutes (by default) ago. There is also an option to pause the script for a set amount of minutes by right clicking the icon in the tray and selecting "pause script".

Hope this helps someone else. Enjoy!


[Autominer.zip](https://github.com/tutulino/Megaminer/files/1755658/Autominer.zip)
